lib/std/tempfiles.nim

```diff
@@ -103,46 +116,60 @@ proc genTempPath*(prefix, suffix: string, dir = ""): string =
   let dir = getTempDirImpl(dir)
   result = dir / (prefix & randomPathName(nimTempPathLength) & suffix)
 
-proc createTempFile*(prefix, suffix: string, dir = ""): tuple[fd: File, path: string] =
-  ## Creates a new temporary file in the directory `dir`.
+proc createTempFile*(prefix, suffix: string, dir = ""): tuple[cfile: File, path: string] =
```

Comment:
```suggestion
proc createTempFile*(prefix, ext: string, dir = ""): tuple[cfile: File, path: string] =
```

Then document that the `ext` will be the file extension of the file, 
so users can pass something like `".csv"` or something.

Reply:
it doesn't have to be file extension, you can pass `suffix = "_baz.txt"`